README.md

```diff
@@ -53,6 +53,8 @@ import { MainBundlePath, DocumentDirectoryPath } from 'react-native-fs'
 
 ***NOTE: only support zip folder, not file entries***
```

Comment:
Maybe we also need to remove this note because we could zip file and [files].

Reply:
hi @plrthink ,I don't know if I'm right, is zip(folder) and zip([file]) supported instead of zip(file)? You mean we should remove this line:

> ***NOTE: only support zip folder, not file entries***

 ? 
I have not tried to compress multiple files so I am not sure if zip([file,file,...]) works

Reply:
yes, we do support it.